applications/gafaelfawr/values-roundtable-dev.yaml

```diff
@@ -0,0 +1,40 @@
+# Use the CSI storage class so that we can use snapshots.
+redis:
+  persistence:
+    storageClass: "standard-rwo"
+
+cloudsql:
+  enabled: true
+  instanceConnectionName: "roundtable-dev-abe2:us-central1:roundtable-dev-572a252e"
+  serviceAccount: "gafaelfawr@roundtable-dev-abe2.iam.gserviceaccount.com"
+
+config:
+  slackAlerts: false
+
+  github:
+    clientId: "083a615e03ca3dde9392"
+
+  # Support OpenID Connect clients like Chronograf.
+  oidcServer:
+    enabled: false
+
+  groupMapping:
+    "admin:provision":
+      - github:
+          organization: "lsst-sqre"
+          team: "square"
+    "exec:admin":
+      - github:
+          organization: "lsst-sqre"
+          team: "square"
```

Comment:
It's an interesting question what group mappings to set up for the Roundtable use case. I think I'd start with just `exec:admin`. I think `admin:provision` is going to go away; it was originally created to talk to moneypenny, which is being removed with the new lab controller work. (I also kind of want to rename `exec:admin` to `admin:<something>` in the future, but we can do that later.)